com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.py

```diff
@@ -62,10 +67,15 @@ def filter(self, func):
 
     def transform(self, func):
         """
-        Transforms each tuple from this stream into 0 or 1 tuples using the supplied function.
+        Transforms each tuple from this stream into 0 or 1 tuples using the supplied callable 'func'.
+        The callable input must be either 
+        1) the name of a top-level function defined using 'def' that takes a single parameter for the tuple, or
```

Comment:
I guess lambda functions can also be 'defined', which we don't support.
Removed the comment 'using def', since we already mention that we don't support lambdas in the spldoc.